### DISCUSSION TITLE
When I use git tab completion, it display confusion

### DISCUSSION THREAD

**Discussion by solomonwzs** [ohmyzsh/ohmyzsh#9854] [Help] [answered]
When I use git command and press tab to completion, it display confusion like this:
<img width="806" alt="image" src="https://user-images.githubusercontent.com/907942/115813785-65e2b180-a426-11eb-871e-20b8c6691414.png">

How should I solve it？thx.

Latest ohmyzsh update? Yes
ZSH Version: [zsh 5.8.1 (x86_64-ArchLinux)]

**Comment by mcornella** [comment, marked as answer]
Try running `clear` and trying again. If that doesn't solve it, please post the contents of your zshrc file.

UPDATE: see https://github.com/ohmyzsh/ohmyzsh/discussions/9854#discussioncomment-1285800 for the fix.

**Reply by solomonwzs** [reply]
I run `clear` and it doesn't work. Here is my [zshrc file](https://github.com/solomonwzs/dotfiles/blob/master/zsh/zshrc), and when I use the same config at other PC running MacOS or Centos, this problem doesn't occurs, thx.

**Reply by mcornella** [reply]
I found that this only happens in your dotfiles repository because the `\t` in your commit messages confuse the completion algorithm. I checked by rewriting the last 20 commit messages and removing the tab characters. This is probably a bug in the completion algorithm though, meaning it should be fixed in zsh.

It would also be a good idea not to write the modified files as the commit subject, but to write them in the body (this is done by writing the first line separated with the rest with an empty line).
